index.html

```diff
@@ -9,14 +9,7 @@
       href="https://www.finn.no/favicon.ico"
       type="image/x-icon"
     />
-    <style>
-      f-docs-template {
-        visibility: hidden;
-      }
-      f-docs-template:defined {
-        visibility: visible;
-      }
-    </style>
+    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/tokens/v1/finn-no.css" />
```

Comment:
Should we use Finn as the base theme here? Or should there be like a `no-brand-theme.css`?

Reply:
There should be a no-brand-theme used here, but I don't know if design has that done yet?

Reply:
We will go ahead with finn for now and change it later. 👍

Reply:
Prob easier to use the finn theme for now, since we know what that is supposed to look like.